course/src/matrix3.cpp

```diff
@@ -90,6 +91,26 @@ double Matrix3::det() const {
   return A - B + C;
 }
 
+Matrix3 Matrix3::product(const Matrix3& r_matrix) const {
+  Matrix3 aux;
+  for (int i = 0; i < 3; ++i) {
+    for (int j = 0; j < 3; ++j) {
+        aux[i][j] = col(i).dot(r_matrix.row(j));
```

Comment:
I think columns and rows are inverted in the dot multiplication. It should be rows of the first matrix times the cols on the second one.

Reply:
Addressed.